### PR TITLE
Feature/about component

### PR DESCRIPTION
**Pull request by talmadge704**
check this out

### CODE REVIEW

File: src/tests/footer.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react';
+import App from './../App';
+
+
+
+test('renders Watch This NavBar that has Home text ', () => {
+  render(<Footer />);
+  const HomeElement = screen.getByText(//i);
```
Comment (tissawd) [inline src/tests/footer.test.js:8]: fill in regex here for getByText

**Review by tissawd — CHANGES_REQUESTED** [review]

File: src/tests/footer.test.js (hunk 1 above)
Comment (ConradC2) [reply src/tests/footer.test.js:8]: Copy will update

Comment (tissawd) [reply src/tests/footer.test.js]: Thanks

**Review by SmoothE21 — APPROVED** [review]
This is dope.

**Review by SmoothE21 — APPROVED** [review]

**Review by ConradC2 — APPROVED** [review]
Approved

**Review by tissawd — APPROVED** [review]

**Pull request merged**